pkg/virt-handler/vm.go

```diff
@@ -586,10 +587,12 @@ func (d *VirtualMachineController) updateVMIStatus(vmi *v1.VirtualMachineInstanc
 						if err != nil {
 							return err
 						}
-						if iface.PodIP != existingInterfaceStatusByName[domainInterface.Alias.Name].IP {
+
+						if iface.PodIP != existingInterfaceStatusByName[domainInterface.Alias.Name].IP ||
```

Comment:
Please keep the order of expected/actual consistent. The original was iface.PodIP != existing... the one you added is the opposite order.

Reply:
done